### ISSUE TITLE
Unittest long_double_representation _INTEL_EXTENDED_12B

### ISSUE THREAD

**Issue by erikjoh** [nicklashersen/numpy#51]
Just create an input that runs this branch https://codecov.io/gh/numpy/numpy/src/master/numpy/core/setup_common.py#L374 and assert the return value. Should not be too difficult.

**Comment by Eika96** [comment]
Just create the file `numpy/core/tests/test_setup_common.py` and add something like:

```python
from __future__ import division, absolute_import, print_function

import pytest

from numpy.core.setup_common import long_double_representation
from numpy.testing import assert_

class TestSetupCommon(object):

    def test_long_double_representation_intel_extended_12b(self):
        """
        Test that a valid INTEL_EXTENDED_12B input gives the correct return.
        """

        lines = ['0 000 000 000 000 001 043 105 147 211 253 315 357',
                 '1 000 000 000 000 240 242 171 353 031 300 000 000',
                 '2 376 334 272 230 166 124 062 020']

        return_value = long_double_representation(lines)

        assert_(return_value == 'INTEL_EXTENDED_12_BYTES_LE')
```

**Comment by erikjoh** [comment]
Solved in #81
